tests/integration/test_backups.py

```diff
@@ -170,7 +170,7 @@ async def test_backup_and_restore(ops_test: OpsTest, cloud_configs: Tuple[Dict,
         # Run the "create backup" action.
         logger.info("creating a backup")
         action = await ops_test.model.units.get(replica).run_action(
-            "create-backup", **{"type": "diff"}
+            "create-backup", **{"type": "differential"}
```

Comment:
A small test for incremental backup would be important.

Reply:
Though I agree, I fear adding it now (i.e. introducing 2 extra heavy operations, incremental backup creation + restore) will make the integration test even more long and unstable due to scaling/wait_for_idle (this happened with differential backups before, causing [this story](https://chat.canonical.com/canonical/pl/qzq5cfq7i3gtbrcmhqg4pobtmy) ) That's why I created https://warthogs.atlassian.net/browse/DPE-4475 as a follow-up: Once the test has improved in terms of stability and/or speed, then we could add it without risk of degrading the CI. LMK what you think.

Reply:
My bad. I forgot about this follow-up you created.